CHANGELOG.rst

```diff
@@ -21,6 +21,7 @@ v0.20.8
 - [ENH] Added ``fill_direction`` function for forward/backward fills on missing values
   for selected columns in a dataframe. @samukweku
 - [ENH] Simpler logic and less lines of code for expand_grid function @samukweku
+- [ENH] Added function ``groupby_topk`` to janitor functions @mphirke
```

Comment:
This should go in the "new version on deck" section. Version 0.20.8 was just released.

Reply:
I changed this but there is a conflict now. I'll fix it in the next commit.

Reply:
Fixed this.